specification/common-types/resource-management/v6/managedidentitywithdelegation.json

```diff
@@ -0,0 +1,54 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Common types",
+    "version": "6.0"
+  },
+  "paths": {},
+  "definitions": {
+    "DelegatedResource": {
+      "type": "object",
+      "description": "Delegated resource properties - internal use only.",
+      "properties": {
+        "resourceId": {
+          "type": "string",
+          "description": "The ARM resource id of the delegated resource - internal use only."
```

Comment:
>The ARM resource id of the delegated resource - internal use only." [](http://example.com/codeflow?start=26&length=67)

what do we call it instead of 'ARM resource id' again? Just 'resource identifier'?

Reply:
in this case we are also using the description of v5: https://github.com/Azure/azure-rest-api-specs/blob/0bbf10474eb460cb35d7e2d3524054418eb19406/specification/common-types/resource-management/v5/managedidentitywithdelegation.json#L15

Reply:
I don't see much downside to improving the style of documentation, but not going to block on this...